Makefile

```diff
@@ -15,8 +15,10 @@ DOC_FILES := \
 	glossary.md
 EPOCH_TEST_COMMIT := 041eb73d2e0391463894c04c8ac938036143eba3
 
-docs: pdf html
+default: docs
+
 .PHONY: docs
+docs: pdf html
```

Comment:
Since you're touching these targets, maybe adjust the `pdf` and `html` targets to actual filenames (`output/docs.pdf`?) or mark them phony as well.  If you go the filename route, you'll want to specify prerequisites for both targets.